### DISCUSSION TITLE
Error: Unexpected end of JSON Input

### DISCUSSION THREAD

**Discussion by TimRoenicke** [livewire/livewire#3299] [7. Bugs (archived)] [answered]
Version: Upgrading 2.5.1 to 2.5.3 (Alpine v2)

I am using livewire-vue as well. 

<img width="486" alt="CleanShot 2021-07-11 at 19 20 27@2x" src="https://user-images.githubusercontent.com/2480028/125212672-12417080-e27d-11eb-9897-f21c3fc94baa.png">

**Comment by joshhanley** [comment, marked as answer]
@TimRoenicke a fix has been tagged for this, please upgrade to Livewire 2.5.5 and let us know if you are still having any issues 🙂

**Reply by TimRoenicke** [reply]
Ya'll rock! Appreciate all you are doing 😄
